src/tests/MapTest.hpp

```diff
@@ -241,8 +235,7 @@ class MapTest : public Test{
         Savitar_core_finalize();
         pthread_mutex_destroy(&snapshot_lock);
 #endif
-        pds::finalize();
-        Persistent::finalize();
+        delete m;
```

Comment:
Are we moving all destructions to the end of the tests?

Reply:
Yes. We didn't do that but we should. We need destruction of `Recoverable` to finish up `EpochSys` instances.